realtime_tools/include/realtime_tools/realtime_thread_safe_box.hpp

```diff
@@ -43,8 +43,10 @@
 #ifndef _WIN32
 #include "realtime_tools/mutex.hpp"
 #define DEFAULT_MUTEX realtime_tools::prio_inherit_mutex
+#define RECURSIVE_MUTEX realtime_tools::prio_inherit_recursive_mutex
 #else
 #define DEFAULT_MUTEX std::mutex
+#define DEFAULT_MUTEX std::recursive_mutex
```

Comment:
```suggestion
#define RECURSIVE_MUTEX std::recursive_mutex
```
Shouldn't this be RECURSIVE_MUTEX?

Sorry for my constant nitpicks